integration_tests/src/main/python/json_test.py

```diff
@@ -21,7 +21,8 @@
 from datetime import timezone
 from conftest import is_databricks_runtime
 from marks import approximate_float, allow_non_gpu, ignore_order
-from spark_session import with_cpu_session, with_gpu_session, is_before_spark_330
+from spark_session import with_cpu_session, with_gpu_session, is_before_spark_330, is_before_spark_340, \
```

Comment:
`is_before_spark_340` was already being pulled in transitively, but better to have it explicitly imported